src/index.ts

```diff
@@ -1,3 +1,5 @@
+require('newrelic');
```

Comment:
import "newrelic"?

Reply:
another possiblity which could lower the chances to send dev data to prod or staging would be to use `node -r newrelic ./dist/index.js` to launch the service in prod/staging
So there is less chance that someone enables it in dev (eg by having a NEW_RELIC env var set to `airlock-prod` ^^) 

But this needs to be defined in the Dockerfile, and the container could also be used for dev ... I don't think it's a good idea to build different versions of the containers, 1 published container should be usable as is in dev, in the community platform or in the prod k8s cluster